docs/quickstart.md

```diff
@@ -55,94 +55,81 @@ sudo cp target/x86_64-unknown-linux-musl/release/{firecracker,jailer} /usr/local
 git clone https://github.com/firecracker-microvm/firecracker-containerd.git
 cd firecracker-containerd
+git submodule update --init --recursive
 sudo DEBIAN_FRONTEND=noninteractive apt-get install -y dmsetup
-make STATIC_AGENT='true'
-sudo cp runtime/containerd-shim-aws-firecracker snapshotter/cmd/{devmapper/devmapper_snapshotter,naive/naive_snapshotter} /usr/local/bin
+make all image
+sudo make install
 
 cd ~
 
-# Download kernel and generic VM image
+# Download kernel
 curl -fsSL -o hello-vmlinux.bin https://s3.amazonaws.com/spec.ccfc.min/img/hello/kernel/hello-vmlinux.bin
-curl -fsSL -o hello-rootfs.ext4 https://s3.amazonaws.com/spec.ccfc.min/img/hello/fsfiles/hello-rootfs.ext4
-
-# Inject the agent, runc, and a startup script into the VM image
-mkdir /tmp/mnt
-# Construct fc-agent.start
-cat >fc-agent.start <<EOF
-#!/bin/sh
-mkdir -p /container
-exec > /container/agent-debug.log # Debug logs from the agent
-exec 2>&1
-touch /container/runtime
-mkdir /container/rootfs
-mount -t auto -o rw /dev/vdb /container/rootfs
-cd /container
-/usr/local/bin/agent -id 1 -debug &
-EOF
-chmod +x fc-agent.start
-truncate --size=+50M hello-rootfs.ext4
-/sbin/e2fsck -f hello-rootfs.ext4
-/sbin/resize2fs hello-rootfs.ext4
-sudo mount hello-rootfs.ext4 /tmp/mnt
-sudo cp $(which runc) firecracker-containerd/agent/agent /tmp/mnt/usr/local/bin
-sudo cp fc-agent.start /tmp/mnt/etc/local.d
-sudo ln -s /etc/init.d/local /tmp/mnt/etc/runlevels/default/local
-sudo ln -s /etc/init.d/cgroups /tmp/mnt/etc/runlevels/default/cgroups
-sudo umount /tmp/mnt
-rmdir /tmp/mnt
 
-cd ~
-
-# Configure containerd to use our new snapshotter
-sudo mkdir -p /etc/containerd
-sudo tee -a /etc/containerd/config.toml <<EOF
+# Configure our firecracker-containerd binary to use our new snapshotter and
+# separate storage from the default containerd binary
+sudo mkdir -p /etc/firecracker-containerd
+sudo mkdir -p /var/lib/firecracker-containerd/containerd
+sudo mkdir -p /run/firecracker-containerd
+sudo tee /etc/firecracker-containerd/config.toml <<EOF
+disabled_plugins = ["cri"]
+root = "/var/lib/firecracker-containerd/containerd"
+state = "/run/firecracker-containerd"
+[grpc]
+  address = "/run/firecracker-containerd/containerd.sock"
 [proxy_plugins]
   [proxy_plugins.firecracker-naive]
     type = "snapshot"
     address = "/var/run/firecracker-containerd/naive-snapshotter.sock"
+
+[debug]
+  level = "debug"
 EOF
 
 cd ~
 
 # Configure the aws.firecracker runtime
+# The long kernel command-line configures systemd inside the Debian-based image
+# and uses a special init process to create a read-write overlay on top of the
+# read-only image.
 sudo mkdir -p /var/lib/firecracker-containerd/runtime
-sudo cp hello-rootfs.ext4 /var/lib/firecracker-containerd/runtime/default-rootfs.img
-sudo cp hello-vmlinux.bin /var/lib/firecracker-containerd/runtime/default-vmlinux.bin
+sudo cp ~/firecracker-containerd/tools/image-builder/rootfs.img /var/lib/firecracker-containerd/runtime/default-rootfs.img
```

Comment:
Maybe we should have the make's `install` rule take care of this? Or provide an dedicated `install_image` target?

Reply:
Possibly, but I'd prefer to do that in a separate PR.